### DISCUSSION TITLE
Create Groups of previously opened PDFs.

### DISCUSSION THREAD

**Discussion by KlingonMike** [sumatrapdfreader/sumatrapdf#3857] [Ideas]
Suggestion: It would be nice if I could create  Named groups so that when I open Sumatra it gives me a menu of the created groups, I then select the one I want, and it opens all the associated PDFs assigned to that group.

**Comment by GitHubRulesOK** [comment]
Hmm that was done a few years ago 
and as a result of multiple problems (actually I used it in working production very well at the time)
It had to be dropped as consuming all the time needed for other fixes and development such as annotations.

Every user (and there are  many) will have a different idea of how such a library system should work for them !

It is possible still to place different start settings in different folders so one starter for novels one starter for comics one for utility bills etc etc.

**Reply by tjdwill** [reply]
Hello,

Bumping this thread as this is a feature I frequently find myself craving. Do you think the re-introduction of groupings would be possible in the future?

Also, if you don't mind answering:

As someone who has not maintained a large project before, how would this feature be represented data structure-wise? What kind of tradeoff(s) would this feature introduce in terms of maintenance?

Thank you for your time.

**Reply by kjk** [reply]
Please open an issue for this.

It can be implemented using advanced settings and remembering list of files under a group name.

**Comment by KlingonMike** [comment]
Hmm, that is sad but your idea does go along with something I am already doing. Thanks for your input.
